editor/components/colors/with-colors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { get } from 'lodash';
+import { get, memoize } from 'lodash';
```

Comment:
We use [memize](https://www.npmjs.com/package/memize) library created by our own @aduth, which is much more performant.

Reply:
Thanks, I've filed #6707 as a follow-up.

Reply:
Thank you for referring the lib and thank you for the rule @tofumatt. Unfortunately, I was not aware of this lib. I will use it in the future.